apps/omg/test/omg/state/persistence_test.exs

```diff
@@ -130,6 +132,38 @@ defmodule OMG.State.PersistenceTest do
              tx |> Transaction.get_inputs() |> hd() |> Utxo.Position.to_input_db_key() |> OMG.DB.spent_blknum()
   end
 
+  @tag fixtures: [:alice]
+  test "collected surpluses are persisted", %{alice: alice} do
+    token1 = <<1::160>>
+    token2 = <<2::160>>
+
+    tx =
+      create_recovered(
```

Comment:
where does create_recovered come from? is it imported? pls declare imports.

Reply:
It's in test helpers from ages ;) I'm open to improve tests, but rather in separate PR :persevere: